tests/lib/rules/no-unnamed-dynamic-imports.js

```diff
@@ -0,0 +1,95 @@
+const {RuleTester} = require('eslint');
+const rule = require('../..//../lib/rules/no-unnamed-dynamic-imports');
+
+const ruleTester = new RuleTester();
+const jsParserInfo = {
```

Comment:
Don’t think you need to use Babel for these, ESLint should be able to handle them natively

Reply:
Babel is still needed for stage 3 `import`s, but I'll pare down the usage.